plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableHandle.java

```diff
@@ -62,6 +63,7 @@ public enum WriteType
     // OPTIMIZE only. Coordinator-only
     private final boolean recordScannedFiles;
     private final Optional<DataSize> maxScannedFileSize;
+    private final Set<DeltaLakeColumnHandle> constraintColumns;
```

Comment:
Can we add a flag to mention that it will be used only for validation.

Reply:
By flag you meant a comment or is there any specific annotation for such cases?

Reply:
Comment added